front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChart.tsx

```diff
@@ -200,9 +234,10 @@ export default function SpeedSpaceChart({
       simulationIsPlaying,
       dispatchUpdateTimePositionValues,
       timeScaleRange,
+      setSharedXScaleDomain,
       additionalAxes
     );
-  }, [chart, additionalAxes]);
+  }, [chart, additionalAxes, sharedXScaleDomain]);
```

Comment:
Oupsi, I think you can remove this also
```suggestion
  }, [chart, additionalAxes]);
```